infra/Pos/Util/LogSafe.hs

```diff
@@ -62,3 +76,40 @@ logMessageS
 logMessageS severity t = execSecureLogWrapped $ do
     name <- getLoggerName
     dispatchMessage name severity t
+
+
+----------------------------------------------------------------------------
+-- Non sensitive buildables
+----------------------------------------------------------------------------
+
+-- | Makes any instance of printing typeclass (e.g. 'Buildable') produce
+-- text without sensitive info.
+newtype NonSensitive a = NonSensitive
```

Comment:
Unfortunately this is a very confusing name. "NonSensitive" means "is not sensitive" or "does not contain sensitive information", but this usage here is really "does contain sensitive information and the sensitive info must be stripped out", which is completely the opposite.

So this needs to be renames from `NonSensitive` to `Sensitive`!